src/test/claimtriebranching_tests.cpp

```diff
@@ -0,0 +1,620 @@
+// Copyright (c) 2015 The LBRY Foundation
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://opensource.org/licenses/mit-license.php
+
+#include "main.h"
+#include "consensus/validation.h"
+#include "consensus/merkle.h"
+#include "primitives/transaction.h"
+#include "miner.h"
+#include "txmempool.h"
+#include "claimtrie.h"
+#include "nameclaim.h"
+#include "coins.h"
+#include "streams.h"
+#include "chainparams.h"
+#include "policy/policy.h"
+#include <boost/test/unit_test.hpp>
+#include <iostream>
+#include "test/test_bitcoin.h"
+
+using namespace std;
+
+
+BOOST_FIXTURE_TEST_SUITE(claimtriebranching_tests, RegTestingSetup)
```

Comment:
Is there a summary of what's tested here? What is tested here that isn't tested in claimtrie_tests.cpp?